frame/clearing-house/src/tests.rs

```diff
@@ -262,7 +262,7 @@ proptest! {
 	#[test]
 	fn mock_vamm_created_id_reflects_genesis_config(vamm_id in any::<Option<VammId>>()) {
 		ExtBuilder { vamm_id , ..Default::default() }.build().execute_with(|| {
-			let created = <Runtime as Config>::Vamm::create(valid_vamm_config());
+			let created = <Runtime as Config>::Vamm::create(&valid_vamm_config());
```

Comment:
I thought this was on `escher` but it seems that it didn't get merged with the last PR

Reply:
I thought that too, (and the `clone` instance above).
I think it was my fault, sorry about that. Still getting used to git when merging different branches and solving merge conflicts.